src/WebJobs.Script/Description/FunctionDescriptorProvider.cs

```diff
@@ -53,9 +54,17 @@ public virtual bool TryCreate(FunctionMetadata functionMetadata, out FunctionDes
                     }
                     else
                     {
-                        binding.Name = DefaultInputParameterName;
+                        ScriptType scriptType = functionMetadata.ScriptType;
+                        binding.Name = scriptType == ScriptType.Powershell ? DefaultPowershellInputParameterName : DefaultInputParameterName;
                     }
                 }
+                else if (functionMetadata.ScriptType == ScriptType.Powershell &&
+                         string.Equals(binding.Name, DefaultInputParameterName, StringComparison.OrdinalIgnoreCase))
```

Comment:
use "input" literal directly here, rather than DefaultInputParameterName since we're explicitly guarding against that. If we change DefaultInputParameterName  in the future, you want this logic to remain unchanged.